<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RenderWidget</class>
 <widget class="QWidget" name="RenderWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>174</width>
    <height>397</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QCheckBox" name="check_renderVertices">
     <property name="text">
      <string>render vertices</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QSlider" name="slider_verticesScaleFactor">
     <property name="value">
      <number>50</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="check_renderEdges">
     <property name="text">
      <string>render edges</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="check_renderFaces">
     <property name="text">
      <string>render faces</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QRadioButton" name="radio_flatShading">
       <property name="text">
        <string>Flat</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">group_faceShading</string>
       </attribute>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radio_phongShading">
       <property name="text">
        <string>Phong</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">group_faceShading</string>
       </attribute>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>156</width>
       <height>161</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="group_faceShading"/>
 </buttongroups>
</ui>
